recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/r/rio-cogeo/rio-cogeo-{{ version }}.tar.gz
-  sha256: "05940d0523b3c0d400177b29a2a33712b84881bbeb82e713a1b9f064422591f9"
+  sha256: 250777ed283811ce6c363e8ef108c382b451465206be68973807bf2ae80cc8f3
```

Comment:
@conda-forge/bot latest `conda-build skeleton` is creating recipes with everything quoted but that seems to break the bot checksum update!